docs/design/architecture_3.0/kata_3.0 _architecture.md

```diff
@@ -0,0 +1,96 @@
+## Overview
+In cloud-native scenarios, there is an increased demand for container startup speed, resource consumption, stability, and security. To achieve this, we would like to introduce you to a solid and secure Rust version kata-runtime. We chose Rust because it is designed to prevent developers from accidentally introducing flaws in their code that can lead to buffer overflows, missing pointer checks, integer range errors, or other memory-related vulnerabilities. Besides, in contrast to Go, Rust makes a variety of design trade-offs in order to obtain the fastest feasible execution performance. 
+Also, we provide the following designs:
```

Comment:
I have a few suggestions about this wording, which I put in branch https://github.com/c3d/kata-containers/commits/kata3.

Reply:
Thanks for your suggestion @c3d , I rebased your commits into one commit, and I add the sub points for this commit. Just want to make the git history looks more clear and clean.